frontend/prebuild/src/app/entity/player.ts

```diff
@@ -49,6 +50,17 @@ export class Player {
 	this.tooltip.visible(isVisible);
   }
   
+  public setStatus(status: string) {
```

Comment:
Do a setter

Reply:
how do

Reply:
`set status(newStatus: string)` then you can just do `player.status = whatever`

Reply:
good to know, however when I try this it breaks the login page (which also uses this object).  Going to skip this comment for now